lib/oli_web/templates/layout/authoring.html.leex

```diff
@@ -75,6 +75,42 @@
       <script type="text/javascript" src="<%= Routes.static_path(@conn, "/js/timezone.js") %>"></script>
     <% end %>
     <script type="text/javascript" src="<%= Routes.static_path(@conn, "/js/dark.js") %>"></script>
+
```

Comment:
Since we are duplicating this code from another layout template, we should refactor it into it's own ts module, like timezone.js and dark.js above

Reply:
I see these comments were marked as resolved, but I don't think the latest commits reflect the changes requested

Reply:
Working on them :) (need to rebase and fix some conflicts as well)

Reply:
Great! Just wanted to make sure they didn't get lost in the dust :)